backend/inspirehep/records/api/literature.py

```diff
@@ -254,10 +261,18 @@ def update_authors_signature_blocks_and_uuids(self):
             )
             return
 
+        redis_url = current_app.config.get("CACHE_REDIS_URL")
+        r = StrictRedis.from_url(redis_url)
+
         for author in self.get("authors", []):
             author_signature_block = signature_blocks.get(author["full_name"])
             if author_signature_block:
                 author["signature_block"] = author_signature_block
+                r.zadd(
+                    "phonetic_blocks",
```

Comment:
``literature_author_phonetic_blocks`` or ``author_phonetic_blocks`` to be more specific